manifests/tekton/tasks/base/cosa-upload-images.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: cosa-upload-images
+spec:
+  steps:
+    - image: 'quay.io/coreos-assembler/coreos-assembler:latest'
+      name: container-build-and-push
+      resources:
+        limits:
+          devices.kubevirt.io/kvm: '1'
+          memory: 4Gi
+        requests:
+          devices.kubevirt.io/kvm: '1'
+          memory: 4Gi
+      script: |
+        #!/usr/bin/env bash
+        set -euxo pipefail
+
+        cd /workspace/coreos
+        cosa push-container  $(params.target-registry) coreos-9:$(params.target-image-tag)
```

Comment:
```suggestion
        cosa push-container  $(params.target-registry):$(params.target-image-tag)
```

Reply:
With @lmzuccarelli , we had a doubt that this didnt look like a regular image format :+1: